ui/src/views/Admin/AddUser.vue

```diff
@@ -21,82 +25,164 @@
 
 <script>
 import axios from "axios";
-
 import Alert from "@/components/Layout/Alert.vue";
 import DynamicForm from "@/components/Form/DynamicForm.vue";
+import { store } from "@/store.js";
 export default {
   components: {
     Alert,
     DynamicForm
   },
-  created() {
+  created () {
     this.config = require("@/config/config.json");
-    this.tempFields = [
-      {
-        id: "username",
-        max: 1,
-        name: "username",
-        required: true,
-        type: "string"
-      },
-      {
-        id: "password",
-        max: 1,
-        name: "password",
-        required: true,
-        type: "password",
-        matching: false
-      },
-      {
-        id: "passwordRepeat",
-        max: 1,
-        name: "password",
-        required: true,
-        type: "password",
-        matching: true
-      }
-    ];
+    if(store.state.isAllowToAccessTheNextPage)
+    {
+      NProgress.start();
```

Comment:
You've got this in here twice. Here and below. Is there some duplicate code that needs to go away?

Reply:
Yes, Done